src/screens/Forum/ForumCategoriesScreen.js

```diff
@@ -1,10 +1,67 @@
-import React from 'react';
-import { View, Text } from 'react-native';
+import React, { useState } from 'react';
+import {
+  ScrollView, Text, View, StyleSheet,
+} from 'react-native';
+import firestore from '@react-native-firebase/firestore';
+import PropTypes from 'prop-types';
+import ForumCategoryCard from './ForumCategoryCard';
+
+const styles = StyleSheet.create({
+  container: {
+    padding: 10,
+    backgroundColor: '#daecf5',
+  },
+});
+
+
+export default function ForumCategoriesScreen({ navigation }) {
+  const [errorMessage, setErrorMessage] = useState(null);
+  const [forumCategories, setForumCategories] = useState([]);
+
+
+  firestore()
+    .collection('forum_categories')
+    .get()
+    .then((snapshot) => {
+      const forumCategoriesData = snapshot.docs.map((doc) => ({
+        ...doc.data(),
+        id: doc.id,
+      }));
+      setForumCategories(
+        forumCategoriesData
+          .sort((a, b) => {
```

Comment:
Let's use the firebase `orderBy` function to sort. Let's do it throughout the forum.